<ui version="4.0" >
<comment> Begin CVS Header </comment>
<comment>   $Source: /Volumes/Home/Users/shoops/cvs/copasi_dev/copasi/UI/StateSubwidget.ui,v $ </comment>
<comment>   $Revision: 1.24.2.1 $ </comment>
<comment>   $Name:  $ </comment>
<comment>   $Author: shoops $ </comment>
<comment>   $Date: 2011/01/26 13:15:48 $ </comment>
<comment> End CVS Header </comment>

<comment> Copyright (C) 2010 by Pedro Mendes, Virginia Tech Intellectual </comment>
<comment> Properties, Inc., University of Heidelberg, and The University </comment>
<comment> of Manchester. </comment>
<comment> All rights reserved. </comment>

 <comment> All rights reserved. </comment>
 <class>StateSubwidget</class>
 <widget class="CopasiWidget" name="StateSubwidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>443</width>
    <height>264</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Expanding" hsizetype="Preferred" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Steady-State Result Window</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" >
     <item>
      <widget class="QLabel" name="topLabel" >
       <property name="text" >
        <string>textLabel1</string>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="mpTabWidget" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex" >
      <number>1</number>
     </property>
     <property name="usesScrollButtons" >
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="mpOptimizationPage" >
      <attribute name="title" >
       <string>Optimization Result</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <item>
        <widget class="Q3TextEdit" name="optimizationResultText" >
         <property name="readOnly" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="mpMetabolitesPage" >
      <attribute name="title" >
       <string>Species</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <item>
        <widget class="Q3Table" name="mpTblMetabolites" >
         <property name="numRows" >
          <number>0</number>
         </property>
         <property name="numCols" >
          <number>7</number>
         </property>
         <property name="readOnly" >
          <bool>true</bool>
         </property>
         <column>
          <property name="text" >
           <string>Name</string>
          </property>
         </column>
         <column>
          <property name="text" >
           <string>Type</string>
          </property>
         </column>
         <column>
          <property name="text" >
           <string>Concentration</string>
          </property>
         </column>
         <column>
          <property name="text" >
           <string>Particle Number</string>
          </property>
         </column>
         <column>
          <property name="text" >
           <string>Rate</string>
          </property>
         </column>
         <column>
          <property name="text" >
           <string>Rate</string>
          </property>
         </column>
         <column>
          <property name="text" >
           <string>Transition Time</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="mpCompartmentsPage" >
      <attribute name="title" >
       <string>Compartments</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <item>
        <widget class="Q3Table" name="mpTblCompartments" >
         <property name="numRows" >
          <number>0</number>
         </property>
         <property name="numCols" >
          <number>4</number>
         </property>
         <property name="readOnly" >
          <bool>true</bool>
         </property>
         <column>
          <property name="text" >
           <string>Name</string>
          </property>
         </column>
         <column>
          <property name="text" >
           <string>Type</string>
          </property>
         </column>
         <column>
          <property name="text" >
           <string>Volume</string>
          </property>
         </column>
         <column>
          <property name="text" >
           <string>Rate</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="mpModelQuantitiesPage" >
      <attribute name="title" >
       <string>Model Quantities</string>
      </attribute>
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <item>
        <widget class="Q3Table" name="mpTblModelValues" >
         <property name="numRows" >
          <number>0</number>
         </property>
         <property name="numCols" >
          <number>4</number>
         </property>
         <column>
          <property name="text" >
           <string>Name</string>
          </property>
         </column>
         <column>
          <property name="text" >
           <string>Type</string>
          </property>
         </column>
         <column>
          <property name="text" >
           <string>Value</string>
          </property>
         </column>
         <column>
          <property name="text" >
           <string>Rate</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="mpReactionsPage" >
      <attribute name="title" >
       <string>Reactions</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <item>
        <widget class="Q3Table" name="mpTblReactions" >
         <property name="numRows" >
          <number>0</number>
         </property>
         <property name="numCols" >
          <number>4</number>
         </property>
         <property name="readOnly" >
          <bool>true</bool>
         </property>
         <column>
          <property name="text" >
           <string>Name</string>
          </property>
         </column>
         <column>
          <property name="text" >
           <string>Flux</string>
          </property>
         </column>
         <column>
          <property name="text" >
           <string>Particle Flux</string>
          </property>
         </column>
         <column>
          <property name="text" >
           <string>Chemical Equation</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="mpJacobianPage" >
      <attribute name="title" >
       <string>Jacobian (Complete)</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2" >
       <property name="margin" >
        <number>0</number>
       </property>
       <item>
        <widget class="QSplitter" name="splitter" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QWidget" name="layoutWidget" >
          <layout class="QGridLayout" >
           <item row="0" column="1" >
            <spacer name="spacer3" >
             <property name="orientation" >
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType" >
              <enum>QSizePolicy::Expanding</enum>
             </property>
             <property name="sizeHint" stdset="0" >
              <size>
               <width>516</width>
               <height>21</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="0" >
            <widget class="QLabel" name="textLabelJacobian" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text" >
              <string>Jacobian (Complete System)</string>
             </property>
             <property name="wordWrap" >
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="2" >
            <widget class="CQArrayAnnotationsWidget" native="1" name="mpJacobianAnnotationWidget" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="layoutWidget" >
          <layout class="QGridLayout" >
           <item row="1" column="0" colspan="2" >
            <widget class="Q3Table" name="tableEigenValues" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="numRows" >
              <number>0</number>
             </property>
             <property name="numCols" >
              <number>2</number>
             </property>
             <property name="readOnly" >
              <bool>true</bool>
             </property>
             <column>
              <property name="text" >
               <string>Real</string>
              </property>
             </column>
             <column>
              <property name="text" >
               <string>Imaginary</string>
              </property>
             </column>
            </widget>
           </item>
           <item row="0" column="1" >
            <spacer name="spacer4" >
             <property name="orientation" >
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType" >
              <enum>QSizePolicy::Expanding</enum>
             </property>
             <property name="sizeHint" stdset="0" >
              <size>
               <width>141</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="0" >
            <widget class="QLabel" name="textLabelEigenvalues" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text" >
              <string>Eigenvalues (Complete System)</string>
             </property>
             <property name="wordWrap" >
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="mpReducedJacobianPage" >
      <attribute name="title" >
       <string>Jacobian (Reduced)</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3" >
       <property name="margin" >
        <number>0</number>
       </property>
       <item>
        <widget class="QSplitter" name="splitter_2" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QWidget" name="layoutWidget" >
          <layout class="QGridLayout" >
           <item row="1" column="0" colspan="2" >
            <widget class="CQArrayAnnotationsWidget" native="1" name="mpJacobianXAnnotationWidget" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item row="0" column="1" >
            <spacer name="spacer3_2" >
             <property name="orientation" >
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType" >
              <enum>QSizePolicy::Expanding</enum>
             </property>
             <property name="sizeHint" stdset="0" >
              <size>
               <width>518</width>
               <height>21</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="0" >
            <widget class="QLabel" name="textLabelJacobianX" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text" >
              <string>Jacobian (Reduced System)</string>
             </property>
             <property name="wordWrap" >
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="layoutWidget" >
          <layout class="QGridLayout" >
           <item row="1" column="0" colspan="2" >
            <widget class="Q3Table" name="tableEigenValuesX" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="numRows" >
              <number>0</number>
             </property>
             <property name="numCols" >
              <number>2</number>
             </property>
             <property name="readOnly" >
              <bool>true</bool>
             </property>
             <column>
              <property name="text" >
               <string>Real</string>
              </property>
             </column>
             <column>
              <property name="text" >
               <string>Imaginary</string>
              </property>
             </column>
            </widget>
           </item>
           <item row="0" column="1" >
            <spacer name="spacer4_2" >
             <property name="orientation" >
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType" >
              <enum>QSizePolicy::Expanding</enum>
             </property>
             <property name="sizeHint" stdset="0" >
              <size>
               <width>141</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="0" >
            <widget class="QLabel" name="textLabelEigenvaluesX" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text" >
              <string>Eigenvalues (Reduced System)</string>
             </property>
             <property name="wordWrap" >
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="mpStabilityPage" >
      <attribute name="title" >
       <string>Stability</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <item>
        <widget class="Q3TextEdit" name="stabilityTextEdit" >
         <property name="readOnly" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="mpProtocolPage" >
      <attribute name="title" >
       <string>Protocol</string>
      </attribute>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <item row="0" column="0" >
        <widget class="Q3TextEdit" name="protocolTextEdit" >
         <property name="readOnly" >
          <bool>true</bool>
         </property>
         <property name="undoRedoEnabled" >
          <bool>false</bool>
         </property>
         <property name="autoFormatting" >
          <set>Q3TextEdit::AutoAll</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <customwidgets>
  <customwidget>
   <class>Q3Table</class>
   <extends>Q3Frame</extends>
   <header>q3table.h</header>
  </customwidget>
  <customwidget>
   <class>Q3TextEdit</class>
   <extends>Q3Frame</extends>
   <header>q3textedit.h</header>
  </customwidget>
  <customwidget>
   <class>CopasiWidget</class>
   <extends>QWidget</extends>
   <header>copasi/UI/copasiWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CQArrayAnnotationsWidget</class>
   <extends>QWidget</extends>
   <header>CQArrayAnnotationsWidget.h</header>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local" >copasi/UI/copasiWidget.h</include>
  <include location="local" >CQArrayAnnotationsWidget.h</include>
 </includes>
 <resources/>
 <connections/>
 <slots>
  <slot>slotSave()</slot>
 </slots>
</ui>
